src/platforms/qurt/eagle_px4_firmware_patch.patch

```diff
@@ -45,12 +45,6 @@ index 4ba2f98..a7b664e 100644
  struct mavlink_logmessage {
  	char text[MAVLINK_LOG_MAXLEN + 1];
  	unsigned char severity;
-diff --git a/src/lib/eigen b/src/lib/eigen
---- a/src/lib/eigen
-+++ b/src/lib/eigen
-@@ -1 +1 @@
--Subproject commit e7850ed81f9c469e02df496ef09ae32ec0379b71
-+Subproject commit e7850ed81f9c469e02df496ef09ae32ec0379b71-dirty
```

Comment:
This doesn't look quite right. Can we get the patched version into the Eigen repo and reference it?